packages/package_image_1_4_1/tool_dependencies.xml

```diff
@@ -0,0 +1,11 @@
+<tool_dependency>
+            <actions>
+                <action type="setup_python_environment">
+                    <repository name="package_image_1_4_1" owner="francescortiz">
```

Comment:
If you want to do this the correct way you really need to use this:
```xml
         <action type="setup_python_environment">
               <repository name="package_python_2_7" owner="iuc">
                   <package name="python" version="2.7" />
               </repository>
               <package">https://codeload.github.com/francescortiz/image/tar.gz/master</package>
         </action>
```
This tells galaxy to setup the python environment use python_2_7, which will be installed for you.